src/handle_action.cpp

```diff
@@ -1367,13 +1367,13 @@ static void fire()
         if( u.has_effect( effect_relax_gas ) ) {
             if( one_in( 8 ) ) {
                 add_msg( m_good, _( "Your willpower asserts itself, and so do you!" ) );
-                reach_attack( u );
+                reach_attack( g->u );
```

Comment:
```suggestion
                reach_attack( u );
```

Reply:
and at the top of this function change `player &u = g->u;` to `avatar &u = g->u;`